ui/js/reducers/claims.js

```diff
@@ -40,20 +40,33 @@ reducers[types.RESOLVE_URI_CANCELED] = function(state, action) {
 }
 
 
-reducers[types.CLAIM_LIST_MINE_STARTED] = function(state, action) {
+reducers[types.FETCH_CLAIM_LIST_MINE_STARTED] = function(state, action) {
   return Object.assign({}, state, {
     isClaimListMinePending: true
   })
 }
 
-reducers[types.CLAIM_LIST_MINE_COMPLETED] = function(state, action) {
-  const myClaims = Object.assign({}, state.myClaims)
-  action.data.claims.forEach((claim) => {
-    myClaims[claim.claim_id] = claim
+reducers[types.FETCH_CLAIM_LIST_MINE_COMPLETED] = function(state, action) {
+  const {
+    claims,
+  } = action.data
+  const myClaims = new Set(state.myClaims)
+  const byUri = Object.assign({}, state.claimsByUri)
+
+  claims.forEach(claim => {
+    const uri = lbryuri.build({
+      contentName: claim.name,
+      claimId: claim.claim_id,
+      claimSequence: claim.nout,
```

Comment:
I think rather than misses, this is going to cause duplicates. So rather than having dupes separated into `claimsByUri` and `myClaims` now we've just moved the duplication problem into the `claimsByUri` object, but a step in the right direction perhaps for removing dupes?

I think perhaps long term it might be better to store claims in `claims.byId` and then map `byUri`, `myClaims`, etc to claim ids.